app/src/main/java/org/wikipedia/travel/MainPlannerFragment.java

```diff
@@ -242,6 +256,37 @@ public void failure(Throwable caught) {
         });
     }
 
+    private void saveDestination() {
```

Comment:
For the new destination history table, are the destinations stored only with the destination name-based constructor? To clarify, the search history displayed will be the same when you edit any of the trips, right?

Reply:
That's correct